### PR TITLE
RGB Light Tiles

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Gives light tiles an RGB colour picker instead of the 6 presets. Also cleans up and refactors the code a little.
wERKwqWm7m

Gif ruins colours, so heres this
image

## Why It's Good For The Game
Why hasn't this been done yet

## Changelog
:cl: AffectedArc07
add: Floor light tiles are now fully RGB 
/:cl:

### CODE REVIEW

**Review by Necaladun — APPROVED** [review]
fix your bullshit blacklights arc this isn't a goth club

**Comment by Necaladun** [comment]
also don't some other maps like the lavaland beach use this? i thought you didn't want map edits, ffs.

**Comment by AffectedArc07** [comment]
Tiles can no longer become goth
image

**Comment by AffectedArc07** [comment]
> also don't some other maps like the lavaland beach use this? i thought you didn't want map edits, ffs.

go sort your ground harness out

**Comment by Wojz** [comment]
> go sort your ground harness out

BODIED

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@
 	gender = PLURAL
 	singular_name = "light floor tile"
 	desc = "A floor tile, made out off glass. Use a multitool on it to change its color."
```
Comment (dearmochi) [inline code/game/objects/items/stacks/tiles/tile_types.dm:109]:
```suggestion
	desc = "A floor tile made of glass. Use a multitool on it to change its color."
```

**Review by dearmochi — APPROVED** [review]
Fine otherwise

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 2]
```diff
@@ -106,8 +106,8 @@
 	name = "light tiles"
 	gender = PLURAL
 	singular_name = "light floor tile"
-	desc = "A floor tile, made out off glass. Use a multitool on it to change its color."
-	icon_state = "tile_light blue"
+	desc = "A floor tile made of glass. Use a multitool on it to change its color."
```
Comment (S34NW) [inline code/game/objects/items/stacks/tiles/tile_types.dm:109]:
```suggestion
	desc = "A floor tile made of glass, with an integrated light. Use a multitool on it to change its color."
```

Nothing major, but it _is_ more than just a glass tile. The rest of the grammar looks fine.

**Review by S34NW — APPROVED** [review]

**Pull request merged**